core/src/test/scala/integration/kafka/tools/TestMirrorMaker.scala

```diff
@@ -0,0 +1,69 @@
+package kafka.tools
+
+import java.util.Properties
+
+import kafka.integration.KafkaServerTestHarness
+import kafka.server.KafkaConfig
+import kafka.tools.MirrorMaker.{MirrorMakerNewConsumer, MirrorMakerProducer}
+import kafka.utils.TestUtils
+import org.apache.kafka.clients.consumer.{ConsumerConfig, KafkaConsumer}
+import org.apache.kafka.clients.producer.{ProducerConfig, ProducerRecord}
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+import org.junit.Assert.assertTrue
+import org.junit.{After, Before, Test}
+
+class TestMirrorMaker extends KafkaServerTestHarness {
```

Comment:
The compiler will complain "MirrorMakerTest is already defined as class MirrorMakerTest".   That's why I changed the name. Do you think there is another way I could avoid this error?